Test/ServerTelemetryChannel.Test/Shared.Tests/Implementation/ErrorHandlingTransmissionPolicyTest.cs

```diff
@@ -219,7 +285,7 @@ private class TestableErrorHandlingTransmissionPolicy : ErrorHandlingTransmissio
         {
             public TimeSpan BackOffTime { get; set; }
 
-            public override TimeSpan GetBackOffTime(NameValueCollection headers = null)
+            protected override TimeSpan GetBackOffTime(NameValueCollection headers = null)
```

Comment:
Since this is a test class it may not be an issue but from a design perspective the property seems to circumvent the helper method.